bastet/src/Test.re

```diff
@@ -1235,6 +1235,146 @@ module Make = (T: TEST, Q: QUICKCHECK with type t = T.test) => {
   };
 };
 
+module JsArray =
```

Comment:
Get rid of this entire module, you don't need to do anything to this file. Test.Array is cross-platform

Reply:
These aren't testing the same module. `Test.JsArray.Functor` is testing `JsArray.Functor` whereas `Test.Array.Functor` is testing `Array.Functor`. If the behavior of one changes, the test of the other doesn't appear to break.

Reply:
Ah I see what you mean... that's annoying. I guess I'll have to add another parameter to that functor for it to reuse the same tests